packages/itwin/property-grid/src/PropertyGridManager.ts

```diff
@@ -3,39 +3,26 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-import { I18N } from "@bentley/imodeljs-i18n";
-import { getClassName, UiError } from "@bentley/ui-abstract";
-import * as i18next from "i18next";
-
-import { PropertyGridManagerFeatureFlags } from "./types";
+import { getClassName, UiError } from "@itwin/appui-abstract";
+import { Localization } from "@itwin/core-common";
+import { LocalizationOptions } from "@itwin/core-i18n";
 
 /**
  * Entry point for static initialization required by various components used in the package.
  * @public
  */
 export class PropertyGridManager {
-  private static _i18n?: I18N;
+  private static _i18n?: Localization;
 
-  /** Feature Flag object with default values */
-  private static _featureFlags: PropertyGridManagerFeatureFlags = {
-    enablePropertyGroupNesting: false,
-  };
   /**
    * Called by IModelApp to initialize PropertyGridManager
    * @param i18n - The internationalization service created by the IModelApp.
    */
-  public static async initialize(
-    i18n: I18N,
-    featureFlags?: PropertyGridManagerFeatureFlags
```

Comment:
Is there no need to have feature flags anymore?

Reply:
i moved this handling into the props of the property grid itself, im not sure why we ever maintained them separately, really ugly